web/src/main/java/org/cbioportal/web/util/LinearDataBinner.java

```diff
@@ -36,26 +36,35 @@ public LinearDataBinner(DataBinHelper dataBinHelper) {
                                            Range<Double> boxRange,
                                            List<Double> values,
                                            Double lowerOutlier,
-                                           Double upperOutlier)
-    {
+                                           Double upperOutlier) {
         Double min = lowerOutlier == null ? Collections.min(values) : Math.max(Collections.min(values), lowerOutlier);
         Double max = upperOutlier == null ? Collections.max(values) : Math.min(Collections.max(values), upperOutlier);
-        
-                
+
         List<DataBin> dataBins = initDataBins(attributeId, min, max, lowerOutlier, upperOutlier);
 
-        // special case for "AGE" attributes
-        if (dataBinHelper.isAgeAttribute(attributeId) &&
-            min < 18 &&
-            (boxRange.upperEndpoint() - boxRange.lowerEndpoint()) / 2 > 18 &&
```

Comment:
@zhx828 @ersinciftci Removing this logic means that we are ignoring the data sets where most (or all) of the values are smaller than 18. When we always use custom (fixed) bins, in certain cases we may end up with a single bin which is `<= 18`. I am not sure fixed data binning is what we always want for the `AGE` attribute.